### ISSUE TITLE
[Release] v0.19.0 Release Candidate Notes

### ISSUE THREAD

**Issue by ysh329** [apache/tvm#17600]
# Introduction

The TVM community has worked since the last release to deliver the following new exciting improvements!

The main tags are below (**bold text is with lots of progress**): Relax, OpenCL, MetaSchedule.

Please visit the full listing of commits for a complete view: [v0.19.dev0...v0.19.0.rc0](https://github.com/apache/tvm/compare/v0.19.dev0...v0.19.0.rc0).

### Community

None.

### RFCs

None.

### Arith
 * [#17469](https://github.com/apache/tvm/pull/17469) - [LLVM]Presburger compile fix for MLIR/LLVM 19.x

### BugFix
 * [#17595](https://github.com/apache/tvm/pull/17595) - [Fix][KVCache] Fix incorrect tile size calculation
 * [#17549](https://github.com/apache/tvm/pull/17549) - [FIX][LLVM] Workaround -mcpu=apple-latest for llvm above 18.0 (#17492)
 * [#17537](https://github.com/apache/tvm/pull/17537) - [FIX][topi.scatter_nd] fixed shape equality assert by using analyzer to prove equality
 * [#17502](https://github.com/apache/tvm/pull/17502) - [FIX][TOPI][strided_slice] Fix topi.strided_slice output shape
 * [#17505](https://github.com/apache/tvm/pull/17505) - [RELAX][ONNX][FIX] add a parser to handle expression in the shape dim names 
 * [#17490](https://github.com/apache/tvm/pull/17490) - [FIX][ONNX][RELAX] Add support for dynamic ShapeExpr in Slice, Squeeze and Flatten
 * [#17467](https://github.com/apache/tvm/pull/17467) - [FIX][RELAX][ONNX] Fix typo in onnx frontend 

### CI
 * [#17596](https://github.com/apache/tvm/pull/17596) - [Test] Skip flaky test to unblock CI
 * [#17451](https://github.com/apache/tvm/pull/17451) - Upgrade CI image to `20241105-030952-3e386fd3`
 * [#17534](https://github.com/apache/tvm/pull/17534) - Upgrade zephyr-sdk to 0.16.9
 * [#17503](https://github.com/apache/tvm/pull/17503) - Upgrade `oneflow==0.9.0`
 * [#17485](https://github.com/apache/tvm/pull/17485) - Revert jax, keras, tensorflow, and tflite upgrades introduced #17425
 * [#17470](https://github.com/apache/tvm/pull/17470) - Pin cpplint==1.6.1

### Docs
 * [#17518](https://github.com/apache/tvm/pull/17518) - Few fixes for broken Adreno docs
 * [#17527](https://github.com/apache/tvm/pull/17527) - Fix typo in TensorIR
 * [#17528](https://github.com/apache/tvm/pull/17528) - Fix Typo in Debugging TVM

### LLVM
 * [#17547](https://github.com/apache/tvm/pull/17547) - Make compilable with LLVM-20
 * [#17538](https://github.com/apache/tvm/pull/17538) - [RUNTIME] Make ORCJIT LLVM executor the default one

### MetaSchedule
 * [#17465](https://github.com/apache/tvm/pull/17465) - Fix a multilevel tiling error on dynamic relax workload

### OpenCL & CLML
 * [#17516](https://github.com/apache/tvm/pull/17516) - [RUNTIME][CLML] Dynamic backward compatibility
 * [#17519](https://github.com/apache/tvm/pull/17519) - [OPENCL][ADRENO] Introduce Qualcomm extension support
 * [#17517](https://github.com/apache/tvm/pull/17517) - [TEST][CLML] Clip test case updated
 * [#17472](https://github.com/apache/tvm/pull/17472) - [Device][OpenCL] add CL_EXEC_STATUS_ERROR_FOR_EVENTS_IN_WAIT_LIST to …

### Relax
 * [#17541](https://github.com/apache/tvm/pull/17541) - Fix bug in convert_layout pass
 * [#17539](https://github.com/apache/tvm/pull/17539) - [KVCache] Fix attention prefill kernel for Metal and Android
 * [#17540](https://github.com/apache/tvm/pull/17540) - Add support for ONNX LPPool
 * [#17536](https://github.com/apache/tvm/pull/17536) - [Frontend][Onnx] Add auto_pad support for conv
 * [#17525](https://github.com/apache/tvm/pull/17525) - support masked_scatter
 * [#17506](https://github.com/apache/tvm/pull/17506) - [Python]Update Rotary positional embedding scaling
 * [#17523](https://github.com/apache/tvm/pull/17523) - Add gather_elements and gather_nd operators
 * [#17511](https://github.com/apache/tvm/pull/17511) - Update ONNX frontend for unique, nonzero and compress
 * [#17509](https://github.com/apache/tvm/pull/17509) - support scatter ops
 * [#17504](https://github.com/apache/tvm/pull/17504) - [ONNX] Add support for dynamic shape expression in Expand
 * [#17482](https://github.com/apache/tvm/pull/17482) - [KVCACHE] Improved schedule for prefill attention
 * [#17445](https://github.com/apache/tvm/pull/17445) - [MetaSchedule] Support CPU weight prepack
 * [#17462](https://github.com/apache/tvm/pull/17462) - Enhance Relax op and ONNX frontend
 * [#17466](https://github.com/apache/tvm/pull/17466) - Revert "[KVCACHE] Improved schedule for prefill attention"

### Runtime
 * [#17557](https://github.com/apache/tvm/pull/17557) - [Dist] Implementation of KV cache transfer
 * [#17498](https://github.com/apache/tvm/pull/17498) - [mrvl]: Support Marvell Hardware Runtime

### TIR
 * [#17423](https://github.com/apache/tvm/pull/17423) - [Schedule] Add annotate_buffer_access primitive

### web
 * [#17545](https://github.com/apache/tvm/pull/17545) - Allows setting powerPreference on webgpu

### Misc
 * [#17593](https://github.com/apache/tvm/pull/17593) - Fix GPU detection in PerStoreFeatureNode
 * [#17554](https://github.com/apache/tvm/pull/17554) - [Refactor] Phase out microTVM
 * [#17542](https://github.com/apache/tvm/pull/17542) - [REFACTOR] Phase out VTA
 * [#17533](https://github.com/apache/tvm/pull/17533) - [Contrib] Remove CLML version print
 * [#17532](https://github.com/apache/tvm/pull/17532) - [3rdparty] Update Picojson with const `operator[]` function (#327)
 * [#17474](https://github.com/apache/tvm/pull/17474) - [TE][CreatePrimFunc] Fix loop carried dependency case with nested block levels
 * [#17501](https://github.com/apache/tvm/pull/17501) - Fix InternalError in StaticPlanBlockMemory when visiting DataflowBlockNode
 * [#17455](https://github.com/apache/tvm/pull/17455) - Compiled with Default Target(LLVM) and Built with USE_MRVL=ON
 * [#17481](https://github.com/apache/tvm/pull/17481) - [Marvell BYOC]: global_max_pool2d and squeeze op support
 * [#17484](https://github.com/apache/tvm/pull/17484) - Replace `np.int` with `np.int32`
 * [#17476](https://github.com/apache/tvm/pull/17476) - Pin pytest-profiling==1.7.0
 * [#17464](https://github.com/apache/tvm/pull/17464) - [JVM] Align Java GraphModule Initialization with Python API
 * [#17458](https://github.com/apache/tvm/pull/17458) - Show the record if the escape sequence is unsupported
